docker/Dockerfile-dev

```diff
@@ -4,7 +4,10 @@
 
 FROM python:3.5-alpine
 
+RUN apk --update --no-cache add \
+	sqlite > /dev/null 2>&1
```

Comment:
I haven't seen the bit about coyping stderr to stdout done in Dockerfiles before, especially for 'apk add'.  Is that a best practice?